controllers/comments-controller.js

```diff
@@ -37,5 +36,16 @@ exports.postComment = (request, response, next) => {
       })
       .catch(next);
   }
-  
+};
+
+exports.deleteComment = (request, response, next) => {
+  const { comment_id } = request.params;
+  return Promise.all([
+    checkExists("comments", "comment_id", comment_id),
+    removeComment(comment_id),
```

Comment:
Nice solution! However, it is worth noting you can also check the rows returned by the delete. If 0, you know nothing has been deleted because the ID doesn't exist.